integrationtests/pom.xml

```diff
@@ -141,12 +141,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
These dependencies where duplicates within this pom, causing warnings in the build.